src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -138,6 +138,7 @@ class FileService(fileRepository: FileRepository,
           hasPreviousPage = currentCursor.isDefined
         ),
         edges,
+        numberOfFilesInFolder,
```

Comment:
I know I named this. But maybe it should be renamed to `totalItems` for consistency?

Reply:
Ah good point yeah, I've renamed it.